partner_relations/model/res_partner_relation_all.py

```diff
@@ -177,6 +177,7 @@ def onchange_type_selection_id(self):
     def write(self, vals):
         """divert non-problematic writes to underlying table"""
         underlying_objs = self.get_underlying_object()
+        underlying_objs = underlying_objs and underlying_objs[0]
```

Comment:
Same that other coment: make self.get_underlying_object() returns a recordset not a list.